languages/wikipron/src/generate_summary.py

```diff
@@ -0,0 +1,76 @@
+import os
+import json
+import csv
+
+from codes import LANGUAGES_PATH, README_PATH, LANGUAGES_SUMMARY_PATH
+
+
+def main():
+    with open(LANGUAGES_PATH, "r") as source:
+        languages = json.load(source)
+    readme_list = []
+    languages_summary_list = []
+    path = "../tsv"
+    for file in os.listdir(path):
+        # Filter out README.md file
+        if file.endswith(".md"):
+            continue
+        with open(f"{path}/{file}", "r") as tsv:
+            num_of_entries = sum(1 for line in tsv)
+        iso639_code = file[: file.index("_")]
+        transcription_level = file[
+            file.rindex("_") + 1: file.index(".")
+        ].capitalize()
+        wiki_name = languages[iso639_code]["wiktionary_name"]
+        if "dialect" in languages[iso639_code]:
+            dialect_key = file[file.index("_") + 1: file.rindex("_")]
+            dialects = languages[iso639_code]["dialect"][dialect_key]
+            if "|" in dialects:
+                dialects = dialects.replace(" |", ",")
+            wiki_name += f" ({dialects})"
+
+        row = [
+            iso639_code,
+            languages[iso639_code]["iso639_name"],
+            wiki_name,
+            str(languages[iso639_code]["casefold"]),
+            transcription_level,
+            str(num_of_entries),
+        ]
+        # TSV and README have different first column.
+        languages_summary_list.append([file] + row)
+        readme_list.append([f"[TSV]({file})"] + row)
+
+    # Sort by wiktionary language name,
+    # with phonemic entries before phonetic
+    def sorting(ele):
```

Comment:
still just use a lambda here.

Reply:
As in: 
```
languages_summary_list.sort(key=lambda ele: ele[3] + ele[5])
readme_list.sort(key=lambda ele: ele[3] + ele[5])
```
Or something along the lines of: 
```
f = lambda ele: ele[3] + ele[5]
languages_summary_list.sort(key=f)
readme_list.sort(key=f)
```
The later is 'DRY-er' but isn't assigning a lambda to a variable an anti-pattern?